control/installation.xml

```diff
@@ -6,42 +6,76 @@ Work around for the text domain
 textdomain="control"
 -->
 
-    <textdomain>control</textdomain>
+<textdomain>control</textdomain>
 
-    <texts>
-      <kvm_host_role>
-        <!-- TRANSLATORS: a label for a system role -->
-        <label>KVM Virtualization Host</label>
-      </kvm_host_role>
-      <kvm_host_role_description>
-        <label>• Kernel-based hypervisor and tools
+<texts>
+  <kvm_host_role>
+    <!-- TRANSLATORS: a label for a system role -->
+    <label>KVM Virtualization Host</label>
+  </kvm_host_role>
+  <kvm_host_role_description>
+      <label>• Kernel-based hypervisor and tools
 • No separate /home partition</label>
-      </kvm_host_role_description>
-    </texts>
+  </kvm_host_role_description>
+</texts>
 
-    <update>
-      <system_roles>
-        <insert_system_roles config:type="list">
-          <insert_system_role>
-            <!-- FATE#317481 -->
-            <system_roles config:type="list">
-              <system_role>
-                <id>kvm_host_role</id>
-                <order config:type="integer">2000</order>
-                <!-- the rest is overlaid over the feature sections and values. -->
-                <partitioning>
-                  <try_separate_home config:type="boolean">false</try_separate_home>
-                </partitioning>
-                <software>
-                  <default_patterns>base kvm_server</default_patterns>
-                  <!-- the cdata trick produces an empty string in the data
-                  instead of omitting the key entirely -->
-                  <optional_default_patterns><![CDATA[]]></optional_default_patterns>
-                </software>
-              </system_role>
-            </system_roles>
-          </insert_system_role>
-        </insert_system_roles>
-      </system_roles>
-    </update>
+<update>
+  <system_roles>
+    <insert_system_roles config:type="list">
+      <insert_system_role>
+        <!-- FATE#317481 -->
+        <system_roles config:type="list">
+          <system_role>
+            <id>kvm_host_role</id>
+            <additional_kernel_parameters/>
+            <ui_mode>simple</ui_mode>
+            <order config:type="integer">2000</order>
+            <enable_autologin config:type="boolean">false</enable_autologin>
+            <enable_firewall config:type="boolean">false</enable_firewall>
+            <firewall_enable_ssh config:type="boolean">false</firewall_enable_ssh>
+            <displaymanager_shutdown>root</displaymanager_shutdown>
```

Comment:
Do you test it? I am pretty sure this won't work as globals parameters like `<additional_kernel_parameters/> <ui_mode>simple</ui_mode> <enable_autologin config:type="boolean">false</enable_autologin> <enable_firewall config:type="boolean">false</enable_firewall><firewall_enable_ssh config:type="boolean">false</firewall_enable_ssh><displaymanager_shutdown>root</displaymanager_shutdown>` lives in `<globals>` section, which is not there.

Reply:
and travis shows I am right, as it does not pass against xml schema - https://travis-ci.org/yast/system-role-kvm/builds/289449709#L386